jiant/proj/main/modeling/model_setup.py

```diff
@@ -17,7 +17,8 @@
 
 
 def setup_jiant_model(
-    model_type: str,
+    model_type,
+    hf_pretrained_model_name: str,
```

Comment:
Worth clarifying why we need both (see below)

Reply:
Good eye. This was an awkward area of the code. I'll take another look at the documentation but the situation is as follows: 
- We can get `model_type` from HF's config object
- We can generate the config object from `hf_pretrained_model_name`
- We cannot get the `model_type` from HF's model object (I need to check this again)
- It seemed awkward to generate a config object with `AutoConfig` every time I wanted `model_type`
It seems like the cleanest way forward it to just pass in `hf_pretrained_model_name` and get the model_type from an AutoClass method.

Reply:
Changed to only use `hf_pretrained_model_name_or_path`